src/org/exist/storage/BrokerPool.java

```diff
@@ -1657,7 +1686,7 @@ void release(final DBBroker broker) {
             //Broker is no more used : inactivate it
             for(final DBBroker inactiveBroker : inactiveBrokers) {
                 if(broker == inactiveBroker) {
-                    LOG.error("Broker is already in the inactive list!!!");
+                    LOG.error("Broker " + broker.getId() + " is already in the inactive list!!!");
```

Comment:
Could have been `LOG.error("Broker {} is already in the inactive list!!!". broker.getId());` ? anyway leave it